cirrus/google_cloud/manager.py

```diff
@@ -956,16 +958,15 @@ def give_service_account_billing_access(
         # get project IAM policy and see if the sa already has the necessary access
         policy = self._get_project_iam_policy(project_id)
         role = GooglePolicyRole(name=full_billing_role_resource)
```

Comment:
i think `role` is not used anywhere, and it's getting overwritten by the `for` loop anyway

Reply:
oh, yup, gets created further down